app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -171,7 +189,7 @@ def create_jdr_report(ems_ref)
 
     def self.raw_alerts_connect(hostname, port, username, password)
       require 'hawkular_all'
-      url         = URI::HTTP.build(:host => hostname, :port => port.to_i, :path => '/hawkular/alerts').to_s
+      url = URI::HTTP.build(:host => hostname, :port => port.to_i, :path => '/hawkular/alerts').to_s
```

Comment:
please keep formatting changes of code that otherwise is not changed as much as possible outside of the PR